resources/views/web-view.php

```diff
@@ -1,15 +1,17 @@
 <?php
+
 use function DT\Home\route_url;
 
 /**
  * @var array $app
  * @var string $url
  */
-$this->layout( 'layouts/web-view' );
+$this->layout('layouts/web-view');
```

Comment:
Your linter removes spaces around the function parameters (line 8 and 14), but there should be spaces according to WordPress CodeStyle. Please add them back.